src/components/common/NameRequestInfo.vue

```diff
@@ -129,33 +130,29 @@
                 <v-icon>mdi-plus</v-icon>
                 <span>Add a Name Translation</span>
               </v-btn>
+
+              <!-- Add Name Translation component -->
+              <div v-if="isAddingNameTranslation" class="mt-6">
+                <AddNameTranslation
+                  :edit-name-translation="editingNameTranslation"
+                  @addTranslation="addNameTranslation($event)"
+                  @cancelTranslation="cancelNameTranslation()"
+                  @removeTranslation="removeNameTranslation(editIndex)"
+                />
+              </div>
+
+              <!-- List Name Translation component -->
+              <div v-if="getNameTranslations && getNameTranslations.length > 0" class="mt-6">
+                <ListNameTranslations
+                  :isAddingNameTranslation="isAddingNameTranslation"
+                  :translationsList="getNameTranslations"
+                  @editTranslation="editNameTranslation($event)"
+                  @removeTranslation="removeNameTranslation($event)"
+                />
+              </div>
```

Comment:
This still logically belongs all in the same column. (The left column is for the labels.)

Note also that I separated the Add and List components so they'd have the correct whitespace before each.